src/test/java/org/takes/rq/RqFormTest.java

```diff
@@ -86,4 +86,49 @@ public void sameInstance() throws IOException {
             Matchers.is(Boolean.TRUE)
         );
     }
+
+    /**
+     * RqForm.Fake can create fake forms with parameters list.
+     * @throws IOException If something goes wrong.
+     */
+    @Test
+    public void createsFormRequestWithParams() throws IOException {
```

Comment:
@carlosmiranda all test methods is using <code>throws IOException </code> declaration, if <code>throws Exception </code> is better we should change all test methods declarations. @yegor256 what do you think ?